obd/protocols/protocol_legacy.py

```diff
@@ -158,55 +154,43 @@ def parse_message(self, message):
                 # now that they're in order, accumulate the data from each frame
 
                 # preserve the first frame's mode and PID bytes (for consistency with CAN)
-                frames[0].data.pop(2) # remove the sequence byte
+                frames[0].data.pop(2)  # remove the sequence byte
                 message.data = frames[0].data
 
                 # add the data from the remaining frames
                 for f in frames[1:]:
-                    message.data += f.data[3:] # loose the mode/pid/seq bytes
+                    message.data += f.data[3:]  # loose the mode/pid/seq bytes
 
         return True
 
 
-
 ##############################################
 #                                            #
 # Here lie the class stubs for each protocol #
 #                                            #
 ##############################################
 
 
-
 class SAE_J1850_PWM(LegacyProtocol):
     ELM_NAME = "SAE J1850 PWM"
     ELM_ID = "1"
-    def __init__(self, lines_0100):
-        LegacyProtocol.__init__(self, lines_0100)
```

Comment:
The constructor's override has been removed because it does not override anything, does exactly the same stuff as `LegacyProtocol`'s constructor, so overriding is not needed.